The patch comes from OpenEmbedded.
Rebased for ppp-2.4.5. Dongxiao Xu <dongxiao.xu@intel.com>

Updated from OE-Classic to include the pcap hunk.
Signed-off-by: Andreas Oberritter <obi@opendreambox.org>

Upstream-Status: Inappropriate [configuration]

Index: ppp-2.4.7/pppd/Makefile.linux
===================================================================
--- ppp-2.4.7.orig/pppd/Makefile.linux
+++ ppp-2.4.7/pppd/Makefile.linux
@@ -120,10 +120,10 @@ CFLAGS   += -DHAS_SHADOW
 #LIBS     += -lshadow $(LIBS)
 endif
 
-ifneq ($(wildcard /usr/include/crypt.h),)
+#ifneq ($(wildcard /usr/include/crypt.h),)
 CFLAGS  += -DHAVE_CRYPT_H=1
 LIBS	+= -lcrypt
-endif
+#endif
 
 ifdef USE_LIBUTIL
 CFLAGS	+= -DHAVE_LOGWTMP=1
@@ -177,10 +177,10 @@ LIBS	+= -ldl
 endif
 
 ifdef FILTER
-ifneq ($(wildcard /usr/include/pcap-bpf.h),)
+#ifneq ($(wildcard /usr/include/pcap-bpf.h),)
 LIBS    += -lpcap
 CFLAGS  += -DPPP_FILTER
-endif
+#endif
 endif
 
 ifdef HAVE_INET6
